docs/design/generics/details.md

```diff
@@ -2004,6 +2004,12 @@ interface NSpacePoint {
 }
 ```
 
+The pattern of an associated constant declaration must be a symbolic binding
+pattern, and unlike other `let` declarations, an associated constant declaration
+cannot have an initializer:
```

Comment:
Unless it the declaration is preceded by the `default` modifier, see #interface-defaults.